app/Models/Mship/Account.php

```diff
@@ -512,4 +514,35 @@ public function __toString()
     {
         return $this->name;
     }
+
+    /**
+     * Return a valid access token for the user or return null if none.
+     *
+     * @return \League\OAuth2\Client\Token\AccessToken
+     * @return null
+     */
+    public function getTokenAttribute()
+    {
+        if ($this->vatsim_access_token === null) {
+            return;
+        } else {
+            $token = new AccessToken([
+                'access_token'  => $this->vatsim_access_token,
+                'refresh_token' => $this->vatsim_refresh_token,
+                'expires'       => $this->vatsim_token_expires,
+            ]);
+
+            if ($token->hasExpired()) {
+                $token = VatsimOAuthController::updateToken($token);
+            }
+
+            $this->update([
+                'vatsim_access_token'  => ($token) ? $token->getToken() : null,
```

Comment:
```suggestion
                'vatsim_access_token'  => $token->getToken() || null,
```
See if that yields the same results

Reply:
Method removed in ecb238f8